### PR TITLE
CAL-69: reorganized managing documentation

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

reorganizes configuration tables in managing documentation

#### Who is reviewing it?

@peterhuffer @jlcsmith 

#### How should this be tested?

- successful build
- verify content and format of docs.

#### What are the relevant tickets?

https://codice.atlassian.net/browse/CAL-69

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+|id
+|String
+|The ID of the NSILI Source
+|NSILI Federated Source
+|true
+
+| Ior File URL
+| iorUrl
+| String
+| The URL of the IOR file to use for the NSILI CORBA Service. Supported URL protocols are 'http://'
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc]: This should have the https://, ftp://, and file:// here too.

File: distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+| null
+| true
+
+| HTTP/S or FTP Username
+| serverUsername
+| String
+| 'https://'
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc]: Looks like something went wrong when transferring this over? Same for `HTTP/S or FTP Password` and `HTTP/S or FTP Timeout` descriptions.

**Comment by peterhuffer** [comment]
A lot of the descriptions in the tables look like they've been mixed up.

File: distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc [hunk 3]
```diff
@@ -0,0 +1,222 @@
+.[[NSILI_Federated_Source]]NSILI Federated Source
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc]: It doesn't look like this NSILI Federated Source table should be in this file based on the file name.

File: distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+| NSILI Access Criteria License Key
+| accessLicenseKey
+| String
+| Swap coordinates for systems that send latitude in the X field of the coordinate.
+| 
+| false
+
+|===
+
+.[[NSILI_Connected_Source]]NSILI Connected Source
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc]: Same thing here - table shouldn't be in this file.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc]: I'll take another pass at it. My script may need some tweaks.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc]: Found the issue, updated tables

File: distribution/docs/src/main/resources/_contents/_tables/conf-NSILI_Connected_Source-table-contents.adoc [hunk 5]
```diff
@@ -0,0 +1,8 @@
+|<<NSILI_Federated_Source,NSILI Federated Source>>
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_tables/conf-NSILI_Connected_Source-table-contents.adoc]: NSILI Federated conf sneaked in here.

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+| Integer
+| Maximum Number of results to cache for each client.
+| 10000
+| true
+
+| Enabled Outgoing Validation
+| outgoingValidationEnabled
+| Boolean
+| Enabled or disabled outgoing DAG validation against mandatory attributes
+| 10000
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc]: This default value should be a Boolean of `false`. The metatype.xml for the NsiliEndpoint has a typo on this @troymohl

File: distribution/docs/src/main/resources/_contents/_tables/conf-NSILI_Connected_Source-table-contents.adoc (hunk 5 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/conf-NSILI_Connected_Source-table-contents.adoc]: fixed.

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc (hunk 6 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc]: fixed.

**Comment by peterhuffer** [comment]
LGTM 👍

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc (hunk 6 above)
Comment (troymohl) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc]: I fixed this as part of the CAL-64 PR

File: catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 7]
```diff
@@ -44,7 +44,7 @@
         <AD
                 description="Enabled or disabled outgoing DAG validation against mandatory attributes"
                 name="Enabled Outgoing Validation" id="outgoingValidationEnabled" required="true" type="Boolean"
-                default="10000"
+                default="false"
```
Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml]: Why did this change?  Is it a result of a merge?

File: distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.nsili.endpoint-table-contents.adoc [hunk 8]
```diff
@@ -0,0 +1,4 @@
+|<<org.codice.alliance.nsili.endpoint,NSILI Endpoint Configuration>>
+|org.codice.alliance.nsili.endpoint
+|null
```
Comment (jlcsmith) [inline distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.nsili.endpoint-table-contents.adoc]: is this supposed to be null?

**Comment by jlcsmith** [comment]
I'm seeing "Unresolved directive in _contents/_nsili/managing-nsili-contents.adoc - include::../_tables/NSILI_Corba_ORB-table-contents.adoc[]" in section 12.3 of the generated HTML.

File: distribution/docs/src/main/resources/_contents/_tables/conf-NITF_Input_Transformer-table-contents.adoc [hunk 9]
```diff
@@ -0,0 +1,4 @@
+|<<NITF_Input_Transformer,Nitf Imaging Input Transformer>>
+|NITF_Input_Transformer
+|null
```
Comment (jlcsmith) [inline distribution/docs/src/main/resources/_contents/_tables/conf-NITF_Input_Transformer-table-contents.adoc]: should this be null?

File: distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.video.stream.mpegts.UdpStreamMonitor-table-contents.adoc [hunk 10]
```diff
@@ -0,0 +1,4 @@
+|<<org.codice.alliance.video.stream.mpegts.UdpStreamMonitor,MPEG-TS UDP Stream Monitor>>
+|org.codice.alliance.video.stream.mpegts.UdpStreamMonitor
+|null
```
Comment (jlcsmith) [inline distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.video.stream.mpegts.UdpStreamMonitor-table-contents.adoc]: Same question here

File: distribution/docs/src/main/resources/_contents/_tables/conf-NITF_Input_Transformer-table-contents.adoc (hunk 9 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/conf-NITF_Input_Transformer-table-contents.adoc]: Nope. fixed.

File: distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.nsili.endpoint-table-contents.adoc (hunk 8 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.nsili.endpoint-table-contents.adoc]: Nope. fixed.

File: distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.video.stream.mpegts.UdpStreamMonitor-table-contents.adoc (hunk 10 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.alliance.video.stream.mpegts.UdpStreamMonitor-table-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
Fixed "unresolved directive" error.

**Comment by jlcsmith** [comment]
LGTM 👍

**Comment by bdeining** [comment]
LGTM

File: distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc (hunk 4 above)
Comment (shaundmorris) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc]: @ricklarsen did you push up the updated tables? @peterhuffer's comment still seems valid based on the latest on the PR.

**Comment by shaundmorris** [comment]
👍  pending last question on the NSILI Federated Source documentation

File: distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc]: I thought I had, but now I have for sure.

**Pull request merged**